internal/storage/v2/grpc/README.md

```diff
@@ -0,0 +1,12 @@
+# gRPC Remote Storage
+
+Jaeger supports a gRPC-based Remote Storage API that enables integration with custom storage backends not natively supported by Jaeger.
+
+To use a remote storage backend, you must deploy a gRPC server at a known address that implements 
```

Comment:
```suggestion
To use a remote storage backend, you must deploy a gRPC server that implements 
```
Not sure what "at a known address" means. May want to mention explicitly that the write path is using OTLP TraceService and may be deployed at a different port.